### PR TITLE
v0.2.0

### PR DESCRIPTION
**Pull request by SC426**
[Upstream](https://github.com/we-like-parsers/pegen)
[Changelog](https://github.com/we-like-parsers/pegen/releases)

### Actions
- Added submodule
- Built recipe from Pypi
- Finished recipe
- Satisfy Linter
- Update Readme.md

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  skip: true  # [py<38]
+  number: 0
+  script: "{{ PYTHON }} -m pip install --no-deps . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - wheel
```
Comment (cbouss) [inline recipe/meta.yaml:24]: requires `toml`

https://github.com/we-like-parsers/pegen/blob/v0.2.0/pyproject.toml#LL47C57-L47C61

**Review by cbouss — COMMENTED** [review]
LGTM, with addition of toml

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml:24]: and `setuptools_scm`

Comment (SC426) [reply recipe/meta.yaml:24]: Added!

**Review by ELundby45 — APPROVED** [review]
LGTM

**Pull request merged**